### DISCUSSION TITLE
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc9 in position 835: invalid continuation byte

### DISCUSSION THREAD

**Discussion by HaiyuDongfang** [shap/shap#3828] [Q&A]
Hi, I used shap to explain results from XGBoost regressor but the error always jumped out. Could you give me some advice to figure it out? Here is my code:

params = {
    "n_estimators":200,
    "learning_rate":0.01,
    "max_depth":6,
    "subsample":0.8,
    "random_state":42
}
model = XGBRegressor(**params).fit(X_train,y_train)

explainer = shap.TreeExplainer(model)
shap_values = explainer.shap_values(X_train)

shap.summary_plot(shap_values, X_test)

**Comment by Changchang001** [comment]
I had the same problem, did you solve it?
